static/app/components/arithmeticBuilder/token/function.tsx

```diff
@@ -154,36 +312,101 @@ function InternalInput({
         })
       );
     });
-  }, [attributesFilter, functionArguments, getFieldDefinition]);
+  }, [attributesFilter, builderFunctionArguments, getFieldDefinition]);
 
-  const items = useAttributeItems({
+  const attributeItems = useAttributeItems({
     allowedAttributes,
     filterValue,
   });
 
-  const shouldCloseOnInteractOutside = useCallback(
-    (el: Element) => !rowRef.current?.contains(el),
-    [rowRef]
-  );
+  const items = useMemo(() => {
```

Comment:
This is a little odd that we compute `attributeItems` no matter what but drop it afterwards 🤔 the cleaner way would be to have 2 separate inputs entirely but that might be excessive.

Reply:
yeah, we can refactor in a follow up